services/venice-server/src/main/java/com/linkedin/venice/listener/ServerStatsContext.java

```diff
@@ -208,9 +208,11 @@ public HttpResponseStatus getResponseStatus() {
   public void setRequestType(RequestType requestType) {
     switch (requestType) {
       case MULTI_GET:
+      case MULTI_GET_STREAMING:
         currentStats = multiGetStats;
         break;
       case COMPUTE:
+      case COMPUTE_STREAMING:
```

Comment:
So we have streaming request hitting Venice Server from fast client now?

Reply:
You mean compute streaming specifically? Not yet but PYMK could be ramping to it in Q4.

Reply:
Got it.